rosdep/python.yaml

```diff
@@ -2668,6 +2668,12 @@ python-mock:
     xenial_python3: [python3-mock]
     yakkety: [python-mock]
     zesty: [python-mock]
+python-monotonic:
+  arch: [python-monotonic]
+  debian: [python-monotonic]
+  fedora: [python-monotonic]
+  gentoo: [dev-python/monotonic]
```

Comment:
I couldn't find that this package exists on Gentoo: https://packages.gentoo.org/packages/dev-python/monotonic
Maybe you could point me to the place?

Reply:
A quick google-search pointed me to it. But perhaps in another flavor, I'll remove.